.github/workflows/release.yml

```diff
@@ -66,6 +68,7 @@ jobs:
       - name: Publish library
         run: |
           cd urbanairship-react-native/
+          yarn prepare
```

Comment:
pretty sure prepare runs before publish

Reply:
I wasn't sure about that, but you're probably right

Reply:
looks like the answer is yes but fyi https://github.com/yarnpkg/yarn/issues/4634